os_net_config/cli.py

```diff
@@ -574,5 +629,33 @@ def get_iface_config(
     return iface_array
 
 
+def minimum_config(provider,
+                   min_config,
+                   no_activate,
+                   root_dir,
+                   noop):
+    if not min_config:
+        logger.warning("minimum_config is not provided")
+        return ExitCode.MINIMUM_CONFIG_FAILED
+
+    logger.info("%s: Running minimum config", provider)
+    ret = config_provider(
```

Comment:
We probably want a way to just purge provider and not run minimal config, in case there is a default route interface that os-net-config never handled. 

That will happen with some CI or QE use cases, and also with pre-provisioned nodes.